graylog2-web-interface/src/routing/AppFacade.jsx

```diff
@@ -12,7 +12,7 @@ const CurrentUserStore = StoreProvider.getStore('CurrentUser');
 
 import 'javascripts/shims/styles/shim.css';
 import 'stylesheets/bootstrap.min.css';
-import 'stylesheets/font-awesome.min.css';
+import 'font-awesome/css/font-awesome.css';
```

Comment:
Shouldn't this be [`font-awesome/css/font-awesome.min.css`](https://github.com/FortAwesome/Font-Awesome/blob/v4.7.0/css/font-awesome.min.css)?